pkg/store/mockdb/mockdb.go

```diff
@@ -46,6 +46,9 @@ type MockState struct {
 	RetrieveAnnouncementMonthInvoked bool
 	UpdateAnnouncementMonthInvoked   bool
 	DeleteAnnouncementMonthInvoked   bool
+	RetrieveActivityMonthInvoked     bool
+	UpdateActivityMonthInvoked       bool
+	DeleteActivityMonthInvoked       bool
```

Comment:
Huh, I didn't realize that the mockdb kept track of all these bools - that must have been a long time ago. I much prefer our call counter pattern that we're using elsewhere!